### ISSUE TITLE
Unicode character problems

### ISSUE THREAD

**Issue by asvetliakov** [asvetliakov/vscode-neovim#503]
@asvetliakov  Some issues for [wellle/targets.vim](https://github.com/wellle/targets.vim). Before 0.0.70+, everything works. After 0.0.70+, text object motion when __unicode__ character present in the same line breaks the motion.

### Bugs after 0.0.70+

Sample code snippets

```jl
y = function(par1)  # Without unicode
ŷ = function(par1)  # With unicode
```

Normal cursor placed on `a` of `par1`, press `via` to select 'inside argument' text object `par1`
- Without unicode, `par1` is selected as expected
- With unicode `ŷ`, `par` is selected without the `1`

Normal cursor placed on `a` of `par1`, press `cia` to change 'inside argument' text object `par1` and cursor positioned inside bracket `... = function(<my_cursor>)`
- Without unicode, `par1` is deleted and cursor positioned inside bracket `y = function(<my_cursor>)` as expected
- With unicode `ŷ`, `par1` is deleted and cursor positioned outside bracket to the left by 1 column `ŷ = function<my_cursor>()`

Similar bugs for other related motion...

### Versions

OS: Windows with WSL2

asvetliakov/vscode-neovim 0.0.75: https://github.com/asvetliakov/vscode-neovim/commit/20cba2206f1fd04db4eaccc716b1028f96411194

<details>
  <summary>Local Windows NVIM v0.5.0-dev+992-gf3a8c930a</summary>

  NVIM v0.5.0-dev+992-gf3a8c930a
  Build type: RelWithDebInfo
  LuaJIT 2.1.0-beta3
  Compilation: C:/Program Files (x86)/Microsoft Visual Studio/2017/Enterpr
  ise/VC/Tools/MSVC/14.16.27023/bin/Hostx86/x64/cl.exe /DWIN32 /D_WINDOWS
  /W3 /MD /Zi /O2 /Ob1 /DNDEBUG /W3 -D_CRT_SECURE_NO_WARNINGS -D_CRT_NONST
  DC_NO_DEPRECATE -DWIN32 -D_WIN32_WINNT=0x0600 -DINCLUDE_GENERATED_DECLAR
  ATIONS -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LE
  VEL=3 -ID:/a/neovim/neovim/build/config -ID:/a/neovim/neovim/src -IC:/pr
  ojects/nvim-deps/usr/include -ID:/a/neovim/neovim/build/src/nvim/auto -I
  D:/a/neovim/neovim/build/include
  Compiled by runneradmin@fv-az152-708
  
  Features: -acl +iconv +tui
  See ":help feature-compile"
  
     system vimrc file: "$VIM\sysinit.vim"
    fall-back for $VIM: "C:/Program Files/nvim/share/nvim"
  
  Run :checkhealth for more inf
</details>

_Originally posted by @kar9222 in https://github.com/asvetliakov/vscode-neovim/issues/467#issuecomment-753542806_

**Comment by asvetliakov** [comment]
@kar9222 let me look

**Comment by asvetliakov** [comment]
not all unicode characters are affected, this works:
`™™™™ = function(par1)`

**Comment by asvetliakov** [comment]
@kar9222 i don't think your example was working before 0.0.70... The `ŷ` character takes 2 columns in vscode, i'm actually first time encountering it.

**Comment by asvetliakov** [comment]
Fixed in 0.0.76
